File: exercises/D108/solution.pas
var
    a, b, c, ans: longint;
    x, y, dummy: char;
begin
    read(a, dummy, x, b, dummy, y, c);
    if ((x = '+') and (y = '+')) then ans := a + b + c;
    if ((x = '+') and (y = '-')) then ans := a + b - c;
    if ((x = '+') and (y = '*')) then ans := a + b * c;
    if ((x = '-') and (y = '+')) then ans := a - b + c;
    if ((x = '-') and (y = '-')) then ans := a - b - c;
    if ((x = '-') and (y = '*')) then ans := a - b * c;
    if ((x = '*') and (y = '+')) then ans := a * b + c;
    if ((x = '*') and (y = '-')) then ans := a * b - c;
    if ((x = '*') and (y = '*')) then ans := a * b * c;
    writeln(ans)
end.

File: exercises/D108/solution.py
a, x, b, y, c = input().split()
a = int(a)
b = int(b)
c = int(c)
if x == '+' and y == '+': ans = a + b + c
if x == '+' and y == '-': ans = a + b - c
if x == '+' and y == '*': ans = a + b * c
if x == '-' and y == '+': ans = a - b + c
if x == '-' and y == '-': ans = a - b - c
if x == '-' and y == '*': ans = a - b * c
if x == '*' and y == '+': ans = a * b + c
if x == '*' and y == '-': ans = a * b - c
if x == '*' and y == '*': ans = a * b * c
print(ans)
